### ISSUE TITLE
Support react-0.13.0

### ISSUE THREAD

**Issue by basvandijk** [meiersi/blaze-react#15]
[react-0.13.0-rc1](http://facebook.github.io/react/blog/2015/02/24/react-v0.13-rc1.html) is released which currently breaks `blaze-react`:

```
Warning: Something is calling a React component directly. Use a factory or JSX instead. See: http://fb.me/react-legacyfactory
rts.js:8587 uncaught exception in Haskell main thread: TypeError: Cannot read property '__reactAutoBindMap' of undefined
rts.js:8587 TypeError: Cannot read property '__reactAutoBindMap' of undefined
    at ReactClass.createClass.Constructor (lib.js:7761)
    at Object.syncRedrawApp (lib.js:19983)
    at h$$foI (out.js:251803)
    at h$mainLoop (rts.js:9521)
    at rts.js:3632
    at runIfPresent (rts.js:3650)
    at onGlobalMessage (rts.js:3691)
rts.js:9301 Uncaught TypeError: Cannot set property 'status' of null
```

**Comment by basvandijk** [comment]
I fixed this in https://github.com/LumiGuide/blaze-react/commit/a08912fc1d12740b71b6b680cb4571ba78e4b8e9. I will try to make a pull request when I have some time.

**Comment by asayers** [comment]
@basvandijk: Great, thanks Bas.

**Comment by asayers** [comment]
This patch doesn't work with reactjs < 0.12, but it seems like there's no harm in upgrading. I bumped the version of reactjs to 0.12.2 and merged a08912f.
